sql/aiven_extras.sql

```diff
@@ -496,6 +496,29 @@ END;
 $$;
 
 
+DROP FUNCTION IF EXISTS aiven_extras.explain_statement(TEXT);
+CREATE FUNCTION aiven_extras.explain_statement(
+    arg_query TEXT,
+    OUT execution_plan JSON
+)
+RETURNS SETOF JSON
+RETURNS NULL ON NULL INPUT
+LANGUAGE plpgsql
+SECURITY DEFINER
```

Comment:
I would expect the explanation from the commit message here as a comment to not let the next reviewer stumble over there missing search path.